src/drivers/github.js

```diff
@@ -225,19 +225,19 @@ class Github {
     const { actions } = octokit(this.token, this.repo);
     let runners = [];
 
-    if (typeof repo !== 'undefined') {
+    if (typeof repo === 'undefined') {
```

Comment:
Why this change? something related to your non OCD? 😄

Reply:
Yes, actually a Pythonic "bad habit" of using posssitive conditions whenever possible. You can count it as part of the OCD I «don't not have» (note the double negation). 😄  Feel free to revert if you don't think it matters.

Reply:
Its fine I support every OCD habit since I have many